posts/2020-09-25-signed-certificate-with-acme-200010.adoc

```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title: "Install a signed certificate with ACME Certificate Authority on Open Liberty 20.0.0.10"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: Install a signed certificate with ACME Certificate Authority on Open Liberty 20.0.0.10 - OpenLiberty.io
+seo-description: With Open Liberty 20.0.0.10 you can now install a signed certificate with ACME Certificate Authority with acmeCA-2.0 plus significant bug fixes
+blog_description: With Open Liberty 20.0.0.10 you can now install a signed certificate with ACME Certificate Authority with acmeCA-2.0 plus significant bug fixes
+---
+= Install a signed certificate with ACME Certificate Authority with acmeCA-2.0
+Jakub Pomykala <https://github.com/jakub-pomykala>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+// tag::intro[]
+
+With Open Liberty 20.0.0.10 you can now install a certificate authority signed certificate using the ACME protocol with the acmeCA-2.0 feature for improved testing or user experience. Also in this release there are a number of significant bug fixes.
```

Comment:
First instance of ACME should expand the acronym ("Automatic Certificate Management Environment")

Reply:
"certificate authority signed certificate" sounds akward.  How about rewarding to "....install a certificate signed by a  Certificate Authority (CA) using the Automatic Certificate Management Environment (ACME) protocol....."

Reply:
Done